src/repository/cert.rs

```diff
@@ -1499,7 +1499,7 @@ impl TbsCert {
     /// Must be present except in self-signed CA certificates where it is
     /// optional. The keyIdentifier field must be present, the other must not
     /// be.
-    fn take_authority_key_identifier<S: decode::Source>(
+    pub fn take_authority_key_identifier<S: decode::Source>(
```

Comment:
Same question: Does it need to be `pub`?

Reply:
`pub(crate)` is enough. Fixed.